Sources/xcproj/PBXProject.swift

```diff
@@ -184,7 +184,7 @@ extension PBXProject: PlistSerializable {
         dictionary["projectRoot"] = .string(CommentedString(projectRoot))
         dictionary["targets"] = PlistValue.array(targets
             .map { target in
-                let targetName = (proj.getCachedReference(target) as? PBXTarget)?.name
+                let targetName = (proj.objects.getReference(target) as? PBXTarget)?.name
```

Comment:
`.nativeTargets`

Reply:
@yonaskolb - What about aggregate targets?

Reply:
True. That means the old implementation was also missing this. Maybe we need a getTarget(reference: String) -> PBXTarget

Reply:
I think it's fine as it is though. This isn't a heavily trafficked line of code, so the performance tradeoff isn't that bad

Reply:
Adding `getTarget(reference: String)` since I thought the same thing :)